src/views/Plan.vue

```diff
@@ -196,16 +200,26 @@ export default class Plan extends Vue {
 
   async lockComments() {
     this.lockCommentLoader = true;
-    this.planData.commentsAreLocked = true;
     const res = await this.updatePlanAction(this.planData);
-    this.lockCommentLoader = !res;
+    if (!res.status) {
+      this.lockCommentErrMessage = res.message;
+    } else {
+      this.planData.commentsAreLocked = true;
+      this.lockCommentErrMessage = "";
+    }
+    this.lockCommentLoader = false;
   }
 
   async unlockComments() {
     this.lockCommentLoader = true;
-    this.planData.commentsAreLocked = false;
     const res = await this.updatePlanAction(this.planData);
-    this.lockCommentLoader = !res;
+    if (!res.status) {
+      this.lockCommentErrMessage = res.message;
+    } else {
+      this.planData.commentsAreLocked = false;
+      this.lockCommentErrMessage = "";
+    }
+    this.lockCommentLoader = false;
   }
```

Comment:
Duplicate code - can be refactored into a single function, such as `switchCommentsLockState`